drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/metadata/ScenarioHeaderMetaData.java

```diff
@@ -75,7 +75,7 @@ public ScenarioHeaderMetaData(String columnId, String columnTitle, String column
 
     public void edit(final GridBodyCellEditContext context) {
         if (readOnly) {
-            throw new IllegalStateException("A read only header cannot be edited");
+            return;
```

Comment:
@jomarko  @manstis  @ederign 
The reason I throw the exception there is because that method should never be invoked if the readOnly flag is true, and that control should be made by caller code **before** invoking this one.
Wording differently, it is meant exactly to raise an exception to make the developer aware that there is an error in the flow. Replacing it with the "return" will just "hide" error of implementations. Please restore the exception and eventually manage it where needed.